CoreBluetoothMock/CBMPeripheralSpec.swift

```diff
@@ -49,6 +49,14 @@ public enum CBMProximity {
         case .outOfRange: return 127
         }
     }
+    
+    /// The amount the proximity should randomly deviate when reporting RSSI
+    public enum Deviation: Int {
```

Comment:
Perhaps, instead of extending `Int`, there could be a computed property reporting the deviation?
You could also add a `custom(_ deviation: Int)` to improve it even more.

Reply:
I like this - it opts for a more internal implementation and maintains a clean public API. 

I added a case for a custom value like you recommended. There is potential for a user to set a deviation of `10000`+ and the RSSI can be something well out-of-range for its associated proximity, but it doesn't affect anything downstream and won't cause the library to crash or anything. I successfully ran multiple tests with a variety of RSSI deviation values. However, the value must be `>= 0` so the `absolute` value is used if the user provides a negative value. I toyed with the idea of using `UInt8` but that just felt clunky especially since `CBProximity.Deviation` is part of the public-facing API.